packages/google_sign_in/google_sign_in_web/lib/google_sign_in_web.dart

```diff
@@ -89,28 +86,14 @@ class GoogleSignInPlugin extends GoogleSignInPlatform {
     return _gisSdkClient!;
   }
 
-  // This method throws if init or initWithParams hasn't been called at some
-  // point in the past. It is used by the [initialized] getter to ensure that
-  // users can't await on a Future that will never resolve.
-  void _assertIsInitCalled() {
-    if (_initCalled == null) {
-      throw StateError(
-        'GoogleSignInPlugin::init() or GoogleSignInPlugin::initWithParams() '
-        'must be called before any other method in this plugin.',
-      );
-    }
-  }
-
   /// A future that resolves when the plugin is fully initialized.
   ///
   /// This ensures that the SDK has been loaded, and that the `init` method
   /// has finished running.
   @visibleForTesting
   Future<void> get initialized {
-    _assertIsInitCalled();
     return Future.wait<void>(<Future<void>>[
       _jsSdkLoadedFuture,
-      _initCalled!.future,
     ]);
```

Comment:
This can become:
```dart
Future<void> get initialized {
  _assertIsInitCalled();
  return _initCalled!.future; // `init()` already awaits `_jsSdkLoadedFuture` so there's no need to wait for it here.
}
```